### DISCUSSION TITLE
Strange popup. What is it? How to reproduce How to get rid of it

### DISCUSSION THREAD

**Discussion by BlackRule** [valinet/ExplorerPatcher#216] [Q&A] [answered]
When I use LAlt+LShift  to switch between languages from time to time this strange popup apperas:
<img width="340" alt="Screenshot_1" src="https://user-images.githubusercontent.com/3146449/139032643-da2a76cd-aa38-43a6-9d41-3195cd2173ad.png">
I also have uxtheme patcher and Penumbra10 custom theme.
 When I catch the popup using  procexp64 it says 
![image](https://user-images.githubusercontent.com/3146449/139035814-15acac68-7b5c-4f7c-8efb-084dad679600.png)
What's the App.AppXtk181tbxbce2qsex02s8tw7hfxa9xb3t.mca ?

**Comment by BlackRule** [comment]
Hmm that can be this https://superuser.com/questions/1455965/shellexperiencehost-exe-adware-removal-solved-was-chrome-notification/1456726#1456726 but it doesn't seem And why it's empty...

**Comment by Gaurav-Original-ClassicShellTester** [comment]


**Reply by BlackRule** [reply]
I'm not sure if it's related but it can be

**Reply by BlackRule** [reply]
And the author is proffesional about windows shortcuts Maybe he knows how  randomly pressing  LAlt and/or LShift can open this popup Or maby someone here stumbles upon same stuff either looking for solution  or willing to share a solution

**Comment by Gaurav-Original-ClassicShellTester** [comment]


**Reply by BlackRule** [reply]
If cared about "Microsoft recommended" ways I'd use the defective win11 taskbar and not use this great project that we are on a page of.

**Reply by Gaurav-Original-ClassicShellTester** [reply]
I am sorry. My intention was only to inform of the new hotkey since on the Taskbar, the new Language indicator (for which the hotkey seems to be Win+Space) is the only one that I could get to appear in Windows 11. The language bar (for which Left Alt+Shift worked) appears but only as a floating toolbar at the top.

**Comment by Anixx** [comment]
The same problem as mine, probably:
https://www.elevenforum.com/t/how-to-disable-notifications.2030/#post-44400
And I also suffer because I cannot assign the right alt for switching languages as I did on Win8.1. When pressing it I also see this rectangle, the language changes to Russian, but not back.

**Reply by BlackRule** [reply]
There they suggest to turn off notifications completely which I don't want to do

**Reply by Anixx** [reply]
Does the language switcher flyout appear properly on your system?

**Reply by BlackRule** [reply]
No. On win11 it either appeaкs for half a seccond or is not shown at all.

**Reply by BlackRule** [reply]
https://user-images.githubusercontent.com/3146449/137474719-98220ff6-b82f-4992-90ec-83427ac156b5.mp4  https://github.com/valinet/ExplorerPatcher/discussions/98

**Reply by Anixx** [reply]
I had the same problem, now it is gone. Also the Wi-Fi flyout did not function the same way, but the problems seem to be different.
First, I enabled some services I had previously disabled, this fixed the Wi-Fi flyout.

But the problem with language switcher seems to be related to ExplorerPatcher. A pre-release version seemed to fix it. Let's wait for the next release.

**Reply by Gaurav-Original-ClassicShellTester** [reply]
Btw what type of install did you use? regsvr32 DLL or usual dxgi.dll in %windir%?

**Reply by BlackRule** [reply]
I used usual dxgi.dll in %windir%

**Comment by Anixx** [comment]
Try this version: https://github.com/valinet/ExplorerPatcher/files/7401064/dxgi.zip

**Reply by BlackRule** [reply]
Where did you get this vesion from? What tag is this? Did you compile and upload it so you have access to this repo?

**Reply by Anixx** [reply]
No, it was uploaded by Valentin in a discussion thread.

**Reply by BlackRule** [reply]
You should've better send link to the discussion thread so that I knew what was discussed... Could you please?

**Reply by Anixx** [reply]
https://github.com/valinet/ExplorerPatcher/discussions/166

**Comment by BlackRule** [comment, marked as answer]
Yeah it was language switcher that is sometimes broken this way. Also win+space doesn't work when it's broken. Explorer restart fixes it.
